### ISSUE TITLE
Grid Configuration: image not found after upload

### ISSUE THREAD

**Issue by m-sanders** [guardian/grid#1465]
I have successfully got a local copy of grid running with passing health checks for all the services except `usage` but I am having issues uploading images.

When I upload an image it successfully ends up in the S3 bucket but fails to appear in grid. I receive the following message from the: API `/images/{uid}` endpoint:
```
{"errorKey":"image-not-found","errorMessage":"No image found with the given id"}
```

What am I missing? Is this something the S3Watcher or usage?

Thanks

**Comment by kenoir** [comment]
@m-sanders not directly an answer but `usage` will likely never work outside of The Guardian as it relies on the data structure of our Content API to extract usage information about images. You can safely never run `usage` unless you are interested in identifying where images have been used in an external document store and are willing to hack the current implementation to do that.

The `S3Watcher` is a service for uploading images to the grid that have been dropped into an AWS S3 bucket (distinct from that used by the other services) and is intended to run as a Lambda function in the AWS environment. As with `usage` (and the soon to be released `usage-updater` it is not relied upon by the other services and only useful if you want the functionality described above.

**tldr:** no it's not likely to be `S3Watcher` or `usage` :)

Can you see evidence of the image being ingested via the `thrall` or `elasticsearch` logs?

**Comment by theefer** [comment]
My suspicion would be that thrall is not running and the message isn't being processed and written into ES, you need to actually hit `http://localhost:9002` in your browser/curl to kick it into action.

**Comment by m-sanders** [comment]
@kenoir Thanks for the overview of those other services.

Looks like I had a bit of a derp moment where one of my Vagrant boxes was already running an instance of Elastic Search on 9200. So I killed that, cleared the SNS queue, restarted thrall and hit the URL and it is all working. Thanks for your speedy replies.

**Comment by theefer** [comment]
Glad to hear!
